src/lib/bootloader/grub2_widgets.rb

```diff
@@ -339,14 +339,65 @@ def store
     end
   end
 
+  # Specific secure boot widget for ppc as we need to handle also
+  # secure boot state in hardware. For details see https://bugzilla.suse.com/show_bug.cgi?id=1206295
+  class SecureBootWidgetPPC < CWM::CustomWidget
+    def initialize
+      textdomain "bootloader"
+
+      self.handle_all_events = true
+      @secure_boot_widget = SecureBootWidget.new(notify: true)
+
+      super
+    end
+
+    def contents
+      VBox(
+        @secure_boot_widget,
+        ReplacePoint(Id(:ppc_sec_label), Empty())
+      )
+    end
+
+    def init
+      @secure_boot_widget.init # ensure it is init before this widget so we can read value
+      handle_label
+    end
+
+    def handle(events)
+      return unless events["ID"] == @secure_boot_widget.widget_id
+
+      handle_label
+      nil
+    end
+
+  private
+
+    def handle_label
+      # no work if already enabled in HMC
+      return if Systeminfo.ppc_secure_boot_active?
+
+      if @secure_boot_widget.value
+        label_widget = Label(_("Enable also in HMC!"))
```

Comment:
What who?!

Without context, I as the user wouldn't be sure who does the HMC enabling: YaST or myself?

Make it more explicit: "Currently disabled in HMC, remember to enable it also there"
Or, if we can't see the HMC value: "Make sure to enable it also in HMC"

Reply:
About the "HMC" term: 

- UAAB!!
- the related code https://github.com/yast/yast-bootloader/blob/master/src/lib/bootloader/systeminfo.rb#L159 should use same naming, so mention "HMC" in a comment there


(Unexplained Abbreviations Are Bad)

Reply:
s/HMC/firmware/

HMC is one way to manage firmware settings on the pSeries platform

We also support PowerVC, and powernv platform.